portal/client/src/scss/stories.scss

```diff
@@ -0,0 +1,9 @@
+.table {
```

Comment:
is this used in the actual project or part of component? 
naming seems to be not proper